translations/pt/com.microsoft.tfs.client.eclipse.feature/feature.properties

```diff
@@ -3,3 +3,161 @@ Feature-Vendor=Microsoft Corporation
 Feature-Description=Ferramentas para funcionamento com o Microsoft Team Foundation Server e o Visual Studio Team Services. Oferece suporte ao Team Foundation Server 2012, 2013, 2015, 2017 e Visual Studio Team Services.
 Feature-Copyright=Copyright (c) 2017, Microsoft Corporation e outros. Todos os direitos reservados.
 Feature-License-Url=license.html
+Feature-License-Text = MICROSOFT SOFTWARE LICENSE TERMS\n\
```

Comment:
I don't think we need to repeat the English license text here. None of other (not en_us) language files have it.